src/components/toolbox/passphraseInput/index.js

```diff
@@ -124,49 +122,71 @@ class passphraseInput extends React.Component {
-          {[...Array(inputsLength)].map((x, i) => (
-            <span key={i} className={`${grid['col-xs-2']}`}>
-              <Input
-                setRef={ref => ref !== null && this.state.focus === i && ref.focus()}
-                placeholder={this.state.focus === i ? '' : i + 1}
-                className={`${this.state.partialPassphraseError[i] || this.state.passphraseIsInvalid || secondPPFeedback !== '' ? 'error' : ''} ${this.state.focus === i ? 'selected' : ''}`}
-                value={values[i] || ''}
-                type={this.state.showPassphrase ? 'text' : 'password'}
-                autoComplete="off"
-                onBlur={this.removeFocusedField}
-                onFocus={this.setFocusedField}
-                onPaste={this.handlePaste}
-                onChange={this.handleValueChange}
-                onKeyDown={this.keyAction}
-                data-index={i}
-              />
-            </span>
-          ))}
-        </div>
-
-        <div className={styles.footerContent}>
+        <div className={styles.wrapper}>
           <label className={`${styles.showPassphrase}`} onClick={this.handleToggleShowPassphrase}>
             <Icon name={iconName} />
-            <span className={`${styles.label}`}>{showPassphrase ? t('Hide passphrase') : t('Show passphrase')}</span>
+            <span className={`${styles.label}`}>{showPassphrase ? t('Hide') : t('Show')}</span>
           </label>
 
-          <Feedback
-            className={styles.errorMessage}
-            show={!!(this.state.validationError || secondPPFeedback !== '')}
-            status={(this.state.validationError || secondPPFeedback !== '') ? 'error' : ''}
-            showIcon
+          <div className={[
+            styles.inputs,
+            focus !== null ? styles.boxSelected : '',
+            partialPassphraseError.length || passphraseIsInvalid || secondPPFeedback !== '' ? styles.boxOnError : '',
+            'passphrase',
+          ].join(' ')}
           >
-            { secondPPFeedback || this.state.validationError }
-          </Feedback>
+            {
+              // eslint-disable-next-line complexity
+              [...Array(inputsLength)].map((x, i) => (
+                <span key={i}>
+                  <span className={[
+                    styles.inputNumber,
+                    partialPassphraseError[i] || passphraseIsInvalid || secondPPFeedback !== '' ? styles.inputNumberError : '',
+                    focus === i ? styles.inputNumberSelected : '',
+                  ].join(' ')}
+                  >
+                    {`${i + 1}. `}
+                  </span>
+                  <Input
+                    setRef={ref => ref !== null && this.state.focus === i && ref.focus()}
+                    placeholder="___________"
+                    className={`${this.state.partialPassphraseError[i] || this.state.passphraseIsInvalid || secondPPFeedback !== '' ? 'error' : ''} ${this.state.focus === i ? 'selected' : ''}`}
```

Comment:
:art: Based on [design](https://projects.invisionapp.com/d/main#/console/18244706/384058920/preview) the error class should depend only on `partialPassphraseError`
```suggestion
                    className={`${this.state.partialPassphraseError[i] ? 'error' : ''} ${this.state.focus === i ? 'selected' : ''}`}
```
💅 And please split the long line